index.html

```diff
@@ -4,7 +4,182 @@
+              <a href="#" class="nav-link"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+          </ul>
+        </div>
+        <div class="header-bottom">
+          <div class="logo"><a href="#">My Logo</a></div>
+          <div class="nav-right">
+            <ul class="main-nav">
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">GC Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="wrapper">
+      <section class="introduction">
+        <span class="quote">
+          “In helping others, we shall help ourselves, for whatever good we give
+          out completes the circle and comes back to us.”
+        </span>
+        <h1 class="goal">
+          Practicing the better exercice of the heart by reaching down and
+          lifting people up.
+        </h1>
+        <div class="ideal">
+          <span
+            >Love is not what we say, love is what we do. As a song is not a
+            song until we sing it. A bell is not a bell until we ring it. In the
+            same way <strong>Love</strong> is not love until we show it
+            away</span
+          >
+          <h3 class="work-hour">Everyday 8am - 5pm(MON) ~ (FRI) 2pm</h3>
+          @ Genuine charity, from all over the places
+        </div>
+      </section>
+      <section class="program" id="program">
+        <h3 class="program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
```

Comment:
- [ ] **Required:** You are required to use more semantic tags for SEO purposes. Kindly change your `div`s to suitable tags.

![image](https://user-images.githubusercontent.com/27709832/163599632-45ed62a6-962c-4094-b1db-2fa1542ce017.png)

Reply:
Haaa!! : my program section is grabbed inside a section tag! and I had to put each orientation (domain) inside a div I don't see at this specific place where I can use article as I don't have any article inside my webpage nor nav tag there because ther'e no links in there? :-1: 

The `article` HTML element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication) [link](https://www.google.com/search?q=when+to+use+article+in+html&sxsrf=APq-WBt8O3DDfggh5R5NbRROy6MtOzh2kQ%3A1650052190389&ei=XsxZYq22F4u94-EP18qbuAw&ved=0ahUKEwit3ay165b3AhWL3jgGHVflBscQ4dUDCA4&uact=5&oq=when+to+use+article+in+html&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEIAEMgYIABAWEB4yBggAEBYQHjIGCAAQFhAeMgYIABAWEB46BwgAEEcQsAM6BwgAELADEEM6BAgjECdKBAhBGABKBAhGGABQzyBY0T1gykNoAXABeACAAbcCiAGxF5IBBTItNS41mAEAoAEByAEJwAEB&sclient=gws-wiz) about the use of article. so couldn't use it right there as there wasn't no self-contained composition @brytebee

Reply:
resolved 👍